cypress/support/index.ts

```diff
@@ -6,3 +6,11 @@ import 'cypress-keyboard-plugin';
 import '@testing-library/cypress/add-commands';
 
 import './commands';
+
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
+Cypress.on('uncaught:exception', err => {
+  /* returning false here prevents Cypress from failing the test */
+  if (resizeObserverLoopErrRe.test(err.message)) {
+    return false;
+  }
+});
```

Comment:
The `ResizeObserver loop limit exceeded` error started happening after upgrading Cypress from `5.6.0` to `9.2.0`. I'm not sure which Cypress version introduced the error. Perhaps later we can find out.